OpenIGTLinkIF/MRML/vtkMRMLIGTLConnectorNode.cxx

```diff
@@ -332,6 +329,28 @@ void vtkMRMLIGTLConnectorNode::vtkInternal::ProcessIncomingDeviceModifiedEvent(
   }
 }
 
+//----------------------------------------------------------------------------
+void vtkMRMLIGTLConnectorNode::vtkInternal::DeviceAboutToReceiveEvent(igtlioDevice* modifiedDevice)
+{
+  if (!modifiedDevice)
+  {
+    vtkErrorWithObjectMacro(modifiedDevice, "Invalid device");
+    return;
+  }
+
+  vtkMRMLNode* modifiedNode = this->GetOrAddMRMLNodeforDevice(modifiedDevice);
```

Comment:
The name of GetOrAddMRMLNodeforDevice is too complicated. The main function is to get a node, adding it is a side effect, which is enough to be described in the documentation, no need to be included in the name. "for" should be capitalized.